src/BreakpointViewer.cpp

```diff
@@ -438,278 +474,365 @@ void BreakpointViewer::setBreakState()
 	runState = false;
 }
 
-void BreakpointViewer::sync()
+void BreakpointViewer::stretchTable(BreakpointData::Type type)
 {
-	auto sa = ScopedAssign(userMode, false);
-	keepUnusedItems();
-
-	int bpRow = bpTableWidget->rowCount();
-	int wpRow = wpTableWidget->rowCount();
-	int cnRow = cnTableWidget->rowCount();
+	if (type == BreakpointData::ALL) {
+		for (auto* table : selector) {
+			// will not stretch without sorting
+			table->setSortingEnabled(true);
+			table->resizeColumnsToContents();
+		}
+		return;
+	}
 
-	if (breakpoints->breakpointCount() != -1) {
-		for (int index = 0; index < breakpoints->breakpointCount(); ++index) {
-			const Breakpoint& bp = breakpoints->getBreakpoint(index);
+	auto* table = selector[type];
+	// will not stretch without sorting
+	table->setSortingEnabled(true);
+	table->resizeColumnsToContents();
+}
```

Comment:
In a few places you have the pattern:
* if type==ALL then loop over all tables and do stuff
* else do (the same) stuff for one specific table.

Is it worth factoring this out? So something like:
```
auto stretchOneTable = [](auto* table) {
    // will not stretch without sorting
    table->setSortingEnabled(true);
    table->resizeColumnsToContents();
};

if (type == BreakpointData::ALL) {
    for (auto* table : selector) {
        stretchOneTable(table);
    }
} else {
    stretchOneTable(selector[table]);
}
```